### ISSUE TITLE
Suggestion with no logged in user throws exception

### ISSUE THREAD

**Issue by anj4u** [nijel/weblate#927]
The suggestion is written, but I get an server error with this traceback:

```
Internal Server Error: /translate/user/office/en_EN/
Traceback (most recent call last):
 File "/home/user/weblate-env/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
   response = wrapped_callback(request, *callback_args, **callback_kwargs)
 File "/home/user/Weblate-2.4/weblate/trans/views/edit.py", line 524, in translate
   this_unit_url, next_unit_url
 File "/home/user/Weblate-2.4/weblate/trans/views/edit.py", line 293, in handle_translate
   go_next = perform_suggestion(unit, form, request)
 File "/home/user/Weblate-2.4/weblate/trans/views/edit.py", line 211, in perform_suggestion
   request,
 File "/home/user/Weblate-2.4/weblate/trans/models/unitdata.py", line 64, in add
   if can_vote_suggestion(user, unit.translation):
 File "/home/user/Weblate-2.4/weblate/trans/permissions.py", line 59, in wrapper
   return func(user, target_object)
 File "/home/user/Weblate-2.4/weblate/trans/permissions.py", line 136, in can_vote_suggestion
   if check_owner(user, project, 'trans.vote_suggestion'):
 File "/home/user/Weblate-2.4/weblate/trans/permissions.py", line 33, in check_owner
   if not project.owners.filter(id=user.id).exists():
AttributeError: 'NoneType' object has no attribute 'id'
```

**Comment by nijel** [comment]
This should be already fixed in git (8ca2286a5d18550eb7c19a0ac62e4b3403750f83), see #882.

**Comment by anj4u** [comment]
Thank you nijel!
